spec/models/course_spec.rb

```diff
@@ -5,22 +5,32 @@
     before do
       @course = create(:course)
     end
-    context 'when there is a section' do
+
+    context 'when a course is updated' do
+      describe '#send_notification' do
+        it 'executes call method with self parameter' do
+          @section.save
```

Comment:
So, calling `#save` actually won't trigger an update unless a field is actually changed. I'm going to merge this anyway though, and we can go back and do a pass over the tests before merging this into staging.